vivarium/processes/multibody_physics.py

```diff
@@ -83,35 +83,34 @@ def daughter_locations(parent_location, parent_values):
 
 
 class Multibody(Process):
-    """
-    A multi-body physics process using pymunk
-
-    To run with animation on set animate: True, and use the TKAgg matplotlib backend:
-    > MPLBACKEND=TKAgg python vivarium/processes/multibody_physics.py
+    """A multi-body physics process that tracks agent locations and simulates collisions
+        and forces.
```

Comment:
The first line of the docstring is used by some tools to briefly describe the class, so it needs to fit on one line. The official docstring convention guide ([PEP 257](https://www.python.org/dev/peps/pep-0257/#id17)) says:

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. The summary line may be on the same line as the opening quotes or on the next line.

For example, what about this?

```suggestion
    """Tracks agent locations and simulates collisions and forces with multi-body physics
```